CIP-1694/README.md

```diff
@@ -560,7 +571,7 @@ The following table details the ratification requirements for each governance ac
 | 5<sub>c</sub>. Protocol parameter changes, technical group        | ✓    | $P_{5c}$ | \-       |
 | 5<sub>d</sub>. Protocol parameter changes, governance group       | ✓    | $P_{5d}$ | \-       |
 | 6. Treasury withdrawal                                            | ✓    | $P_6$    | \-       |
-| 7. Info                                                           | ✓    | $P_7$    | \-       |
+| 7. Info                                                           | ✓    | $P_7$    | $Q_7$    |
```

Comment:
This is a little awkward, since those thresholds should always be 100%. Otherwise you lose the ability to polling above the threshold (and it doesn't matter that it will never be enacted). Maybe we should actually just say 100% in the table and explain the reasoning.